mkdocs.yml

```diff
@@ -421,6 +421,8 @@ nav:
               - 'Suspend Mechanism Switching S0ix/S3': unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md
               - 'Block boot when battery is low': unified-test-documentation/dasharo-compatibility/359-battery-boot-block.md
               - 'Power State after Power Fail': unified-test-documentation/dasharo-compatibility/360-power-after-fail.md
+              - 'DMI editing tool': unified-test-documentation/dasharo-compatibility/362-coreboot-customizer.md
+
```

Comment:
```suggestion
```

Reply:
Fixed in [recent commit](https://github.com/Dasharo/docs/commit/2f2c60da5365e1c65115c59cf2a341885221995f)